lib/hanami/rspec/generators/helper.rb

```diff
@@ -6,7 +6,4 @@
 ENV["HANAMI_ENV"] ||= "test"
 require "hanami/prepare"
 
-require_relative "support/rspec"
-require_relative "support/features"
-require_relative "support/operations"
-require_relative "support/requests"
+Dir["./spec/support/**/*.rb"].each { |f| require f }
```

Comment:
Mind changing it to this? I like this a bit better for making it clear that requires are for support files within the spec root. And while I haven't tested it, this might actually make these requires more robust if for any reason the cwd is not actually the root of the app.

```suggestion
Dir[SPEC_ROOT.join("support/**/*.rb")].each { |f| require f }
```